frontend/src/metabase/common/components/Pickers/CollectionPicker/hooks.ts

```diff
@@ -26,6 +32,16 @@ const personalCollectionsRoot: CollectionPickerItem = {
   below: ["collection"],
 };
 
+const tenantSpecificCollectionsRoot: CollectionPickerItem = {
+  ...TENANT_SPECIFIC_COLLECTIONS,
+  can_write: false,
+  model: "collection",
+  location: "/",
+  description: "",
+  here: ["collection"],
+  below: ["collection"],
```

Comment:
this might cause bugs with the questionPicker, it might not let you navigate into this folder since it thinks there's no data there